Library/Formula/znapzend.rb

```diff
@@ -0,0 +1,27 @@
+class Znapzend < Formula
+  desc "zfs backup with remote capabilities and mbuffer integration"
+  homepage "http://www.znapzend.org"
+  url "https://github.com/oetiker/znapzend/releases/download/v0.15.3/znapzend-0.15.3.tar.gz"
+  sha256 "27f7c218f4e71beac7b2ba52386a7742506d45c0b3fd40104f44ccef0f373b6e"
+
+  depends_on "perl" if MacOS.version <= :mavericks
+
+  def install
+    system "./configure", "--disable-silent-rules",
+                          "--prefix=#{prefix}"
+    system "make", "install"
+  end
+
+  test do
+    fake_zfs = testpath/"zfs"
+    fake_zfs.write <<-EOS.undent
+      #!/bin/sh
+      for word in "$@"; do echo $word; done >> znapzendzetup_said.txt
+      exit 0
+    EOS
+    chmod 0755, fake_zfs
+    ENV.prepend_path "PATH", testpath
+    assert_match /if you have modified your configuration/, pipe_output("#{bin}/znapzendzetup list 2>&1")
```

Comment:
do you mean like: 
````
system("#{bin}/znapzendzetup list 2>&1")
assert_equal <--EOS.undent
  list -H -o name -t filesystem,volume
EOS, File.read(testpath/"znapzendzetup_said.txt")
````

because that doesn't work for me. It always complains about unexpected \n in the `list -H -o name -t filesystem,volume` line